content/applications/hr/fleet/configuration.rst

```diff
@@ -0,0 +1,234 @@
+=============
+Configuration
+=============
+
+The following document goes through all of the configurations and settings for the *Fleet*
+application, including settings, manufacturers, models, and categories.
+
+Settings
+========
+
+To access the settings menu, go to :menuselection:`Fleet application --> Configuration -->
+Settings`. There are only two settings that need configuration, the vehicle contract alert, and
+a new vehicle request.
+
+End Date Contract Alert
+-----------------------
+
+To help ensure no contracts lapse or vehicles are left without contracted drivers, Odoo has an
+alert in place to let the person responsible for the vehicle contract know when a contract is abut
+to expire. Enter the number of days before a contract is going to expire that an alert email will be
+sent.
+
+.. note::
+   To determine who the responsible person is for a contract, open an individual contract. The
+   person listed as :guilabel:`Responsible` under the :guilabel:`Contract Information` section of
+   the contract is the person who will receive the alert.
+
+   To access all contracts, navigate to :menuselection:`Fleet app --> Fleet --> Contracts` and all
+   contracts appear in the list. Click on a :guilabel:`Contract` to view it.
+
+   An individual contract can also be found by navigating to :menuselection:`Fleet app --> Fleet -->
+   Fleet` and clicking on an individual vehicle. On the vehicle form, click the
+   :guilabel:`Contracts` smart button at the top of the page. The contract(s) associated with this
+   vehicle only appears in the list. Click on an individual contract to open it. The
+   :guilabel:`Responsible` person is listed on the contract.
+
+New Vehicle Request
+-------------------
+
+Set a limit on how many vehicle are in the fleet by setting a limit on requests. When using the
```

Comment:
This is confusing.... the page says: 
`New Vehicle Request
An employee cannot request a new car if there are more than 
1,000
 existing cars availables.`
```suggestion
Set a limit on how many vehicles are in the fleet by setting a limit on requests. When using the
```

Doesn't say limit the fleet size-- but only deter folks from requesting new cars

Reply:
True- I guess I saw this as a way to limit the size of the fleet, since people would not be able to request new cars if X old cars are already available - but I can see how that is a leap.